### PR TITLE
Solution to Numero Uno

### PR DESCRIPTION
**Pull request by polyglotyeoja**
:)

soo......I used a while loop to call console.log 7 times and concatenated what was inside of my var x with '#'.

### CODE REVIEW

File: week1/day1/1.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var console = require('./lib/consolelog.js');
-
-module.exports = function triangle() {
-  for (var i = 1; i <= 7; i++) {
-    console.log('#'.repeat(i))
-  }
+module.exports = function triangle(){
+	var count = 0
+	var x = '#'
+	while(count<7) {console.log(x); x+= '#';count++}
```
Comment (aarondancer) [inline week1/day1/1.js]: A for loop would be better :P

File: week1/day1/2.js [hunk 2]
```diff
@@ -1,5 +1,19 @@
 var console = require('./lib/consolelog.js');
 
 module.exports = function fizzBuzz() {
-  // write your code here
+
+	for(x = 0; x < 100; x++)
+	{
+		if ((x % 5 == 0) && (x % 3 == 0))
```
Comment (jophyson93) [inline week1/day1/2.js]: use "===" instead of "==".

File: week1/day1/1.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-module.exports = function triangle() {
-  for (var i = 1; i <= 7; i++) {
-    console.log('#'.repeat(i))
-  }
+module.exports = function triangle(){
+	for(x=''; x<'#######'; x+='#')
+	{
+		console.log(x)
+	}
 }
```
Comment (ErkanSen14) [inline week1/day1/1.js]: Almost, except your program prints out an extra empty line before your pyramid.

File: week1/day1/2.js [hunk 4]
```diff
@@ -1,5 +1,16 @@
 var console = require('./lib/consolelog.js');
-
 module.exports = function fizzBuzz() {
-  // write your code here
+
+    for (x = 1; x < 101; x++) {
+        if ((x % 3 == 0) && (x % 5 == 0))
```
Comment (ErkanSen14) [inline week1/day1/2.js]: Use `===` instead of `==`.

File: week1/day1/2.js [hunk 5]
```diff
@@ -1,5 +1,16 @@
...
+            console.log("FizzBuzz")
```
Comment (ErkanSen14) [inline week1/day1/2.js]: Double quote heathen >:(

File: week1/day1/3.js [hunk 6]
```diff
@@ -1,5 +1,23 @@
 var console = require('./lib/consolelog.js');
 
-module.exports = function chessboard() {
-  // write your code here
+module.exports = function chessboard(size) {
+var grid = ''
+for (x = 0; x < size; x++) {
+    for (y = 0; y < size; y++) {
+        if (x % 2 == 0) {
```
Comment (ErkanSen14) [inline week1/day1/3.js]: Use `===` instead of `==` yo

**Pull request closed without merging**